base-image/packages.txt

```diff
@@ -0,0 +1,279 @@
+babel==2.14.0
+beautifulsoup4==4.12.3
+bleach==6.1.0
+bokeh==3.4.1
+boto3==1.34.106
+boto3-stubs==1.34.108
+botocore==1.34.106
+botocore-stubs==1.34.94
+brotli-python==1.1.0
+bzip2==1.0.8
+c-ares==1.28.1
+ca-certificates==2024.2.2
+cached-property==1.5.2
+cached_property==1.5.2
+certifi==2024.2.2
+cffi==1.16.0
+charset-normalizer==3.3.2
+click==8.1.7
+cloudpickle==3.0.0
+colorama==0.4.6
+comm==0.2.2
+contourpy==1.2.1
+cytoolz==0.12.3
+dask==2024.5.0
+dask-core==2024.5.0
+dask-expr==1.1.0
+dask-labextension==7.0.0
+debugpy==1.8.1
+decorator==5.1.1
+defusedxml==0.7.1
+distributed==2024.5.0
+entrypoints==0.4
+exceptiongroup==1.2.0
+executing==2.0.1
+fqdn==1.5.1
+freetype==2.12.1
+frozenlist==1.4.1
+fsspec==2024.5.0
+gflags==2.2.2
+gitdb==4.0.11
+glog==0.7.0
+greenlet==3.0.3
+h11==0.14.0
+h2==4.1.0
+hpack==4.0.0
+httpcore==1.0.5
+httpx==0.27.0
+hyperframe==6.0.1
+idna==3.7
+importlib-metadata==7.1.0
+importlib_metadata==7.1.0
+importlib_resources==6.4.0
+ipykernel==6.29.3
+ipython==8.24.0
+isoduration==20.11.0
+jedi==0.19.1
+jinja2==3.1.4
+jmespath==1.0.1
+json5==0.9.25
+jsonpointer==2.4
+jsonschema==4.22.0
+jsonschema-specifications==2023.12.1
+jsonschema-with-format-nongpl==4.22.0
+jupyter-lsp==2.2.5
+jupyter-server-mathjax==0.2.6
+jupyter-server-proxy==4.1.2
+jupyter_client==8.6.1
+jupyter_core==5.7.2
+jupyter_events==0.10.0
+jupyter_server==2.14.0
+jupyter_server_terminals==0.5.3
+jupyterlab==4.2.0
+jupyterlab-bxplorer==0.1.24
+jupyterlab-git==0.50.1
+jupyterlab-s3-browser==0.12.0
+jupyterlab_pygments==0.3.0
+jupyterlab_server==2.27.1
+jupytext==1.16.2
+keyutils==1.6.1
+krb5==1.21.2
+lcms2==2.16
+ld_impl_linux-64==2.40
+lerc==4.0.0
+libabseil==20240116.2
+libarrow==16.0.0
+libarrow-acero==16.0.0
+libarrow-dataset==16.0.0
+libarrow-substrait==16.0.0
+libblas==3.9.0
+libbrotlicommon==1.1.0
+libbrotlidec==1.1.0
+libbrotlienc==1.1.0
+libcblas==3.9.0
+libcrc32c==1.1.2
+libcurl==8.7.1
+libdeflate==1.20
+libedit==3.1.20191231
+libev==4.33
+libevent==2.1.12
+libexpat==2.6.2
+libffi==3.4.2
+libgcc-ng==13.2.0
+libgfortran-ng==13.2.0
+libgfortran5==13.2.0
+libgomp==13.2.0
+libgoogle-cloud==2.23.0
+libgoogle-cloud-storage==2.23.0
+libgrpc==1.62.2
+libjpeg-turbo==3.0.0
+liblapack==3.9.0
+libnghttp2==1.58.0
+libnsl==2.0.1
+libopenblas==0.3.27
+libparquet==16.0.0
+libpng==1.6.43
+libprotobuf==4.25.3
+libre2-11==2023.09.01
+libsodium==1.0.18
+libsqlite==3.45.3
+libssh2==1.11.0
+libstdcxx-ng==13.2.0
+libthrift==0.19.0
+libtiff==4.6.0
+libutf8proc==2.8.0
+libuuid==2.38.1
+libwebp-base==1.4.0
+libxcb==1.15
+libxcrypt==4.4.36
+libzlib==1.2.13
+locket==1.0.0
+lz4==4.3.3
+lz4-c==1.9.4
+markdown-it-py==3.0.0
+markupsafe==2.1.5
+matplotlib-inline==0.1.7
+mdit-py-plugins==0.4.1
+mdurl==0.1.2
+mistune==3.0.2
+msgpack-python==1.0.8
+multidict==6.0.5
+mypy-boto3-s3==1.34.105
+nbclient==0.10.0
+nbconvert==7.16.4
+nbconvert-core==7.16.4
+nbdime==4.0.1
+nbformat==5.10.4
+ncurses==6.5
+nest-asyncio==1.6.0
+notebook==7.2.0
+notebook-shim==0.2.4
+numpy==1.26.4
```

Comment:
at some point we need to bump this to 2.0, and this migration feels to be a good point for it.

Reply:
For now, I think the hold back is lsdb notebooks, and lightkurve, which which has a some dependency that requires numpy<2

Reply:
yes, they rely on healpy atm, and also one of the ML notebooks also has a dependency that requires np <2. So I would suggest having a reminder issue about numpy 2, and just roll with the old one for now.